includes/class-newspack-popups-model.php

```diff
@@ -642,17 +667,20 @@ public static function generate_inline_popup( $popup ) {
 		$classes[]            = ( ! empty( $popup['title'] ) && $display_title ) ? 'newspack-lightbox-has-title' : null;
 		$classes[]            = $is_newsletter_prompt ? 'newspack-newsletter-prompt-inline' : null;
 
-		add_filter(
-			'newspack_analytics_events',
-			function ( $evts ) use ( $popup, $body, $element_id ) {
-					return array_merge( $evts, self::get_analytics_events( $popup, $body, $element_id ) );
-			}
-		);
+		$analytics_events = self::get_analytics_events( $popup, $body, $element_id );
+		if ( ! empty( $analytics_events ) ) {
+			add_filter(
+				'newspack_analytics_events',
+				function ( $evts ) use ( $analytics_events ) {
+					return array_merge( $evts, $analytics_events );
+				}
+			);
+		}
 
 		ob_start();
 		?>
 			<?php self::insert_event_tracking( $popup, $body, $element_id ); ?>
-			<amp-layout amp-access="popup_<?php echo esc_attr( $popup['id'] ); ?>.displayPopup" amp-access-hide class="<?php echo esc_attr( implode( ' ', $classes ) ); ?>" role="button" tabindex="0" style="<?php echo esc_attr( self::container_style( $popup ) ); ?>" id="<?php echo esc_attr( $element_id ); ?>">
+			<amp-layout <?php echo self::get_access_attrs( $popup ); ?> class="<?php echo esc_attr( implode( ' ', $classes ) ); ?>" role="button" tabindex="0" style="<?php echo esc_attr( self::container_style( $popup ) ); ?>" id="<?php echo esc_attr( $element_id ); ?>">
```

Comment:
Can the access attributes be escaped?

Reply:
No (it's HTML, so it should not be escaped), but the `get_access_attrs` is safe (so far!)